src/screen.cpp

```diff
@@ -43,6 +43,29 @@ std::map<GLFWwindow *, Screen *> __nanogui_screens;
 static bool gladInitialized = false;
 #endif
 
+inline NVGcontext* nvgCreateContext(int flags) {
+#if defined NANOVG_GL2_IMPLEMENTATION
```

Comment:
Can you change this syntax to ``if defined(NANOVG_GL2_IMPLEMENTATION)`` to match the rest of the codebase? (elsewhere as well)